frontend/e2e-tests/MoOrganisationUnitTerminate.js

```diff
@@ -0,0 +1,38 @@
+import { Selector } from 'testcafe'
+import { baseURL } from './support'
+
+let moment = require('moment')
+
+fixture('Organisation test')
+  .page(`${baseURL}/organisation`)
+
+const dialog = Selector('#orgUnitTerminate')
+
+const parentInput = dialog.find('input[data-vv-as="Enhed"]')
+
+const fromInput = dialog.find('.from-date input.form-control')
+
+test('Workflow: terminate employee', async t => {
+  let today = moment()
+
+  await t
+    .setTestSpeed(0.8)
+    .hover('#mo-workflow', {offsetX: 10, offsetY: 130})
+    .click('.btn-unit-terminate')
+
+    .expect(dialog.exists).ok('Opened dialog')
+
+    .click(parentInput)
+    .click(dialog.find('li .item .link-color'))
+
+    .click(fromInput)
+    .hover(dialog.find('.vdp-datepicker .day:not(.blank)')
+           .withText(today.date().toString()))
+    .click(dialog.find('.vdp-datepicker .day:not(.blank)')
+           .withText(today.date().toString()))
+    .expect(fromInput.value).eql(today.format('DD-MM-YYYY'))
+
+    .click(dialog.find('.btn-primary'))
+
+    .expect(Selector('#orgUnitTerminate').exists).notOk()
```

Comment:
You should probably check the results of the termination as well — i.e. that the unit lifetime actually changed to the expected values.